plugins/paytovote/paytovote.go

```diff
@@ -0,0 +1,183 @@
+package paytovote
+
+import (
+	"fmt"
+
+	abci "github.com/tendermint/abci/types"
+	"github.com/tendermint/basecoin/types"
+	"github.com/tendermint/go-wire"
+)
+
+const (
+	TypeByteCreateIssue byte = 0x00
+	TypeByteVoteFor     byte = 0x01
+	TypeByteVoteAgainst byte = 0x02
+	TypeByteVoteSpoiled byte = 0x03
+)
+
+type P2VPluginState struct {
+	TotalCost    types.Coins
+	Issue        string
+	votesFor     int
+	votesAgainst int
+	votesSpoiled int
+}
+
+type P2VTx struct {
+	Valid            bool
+	Issue            string      //Issue being voted for
+	ActionTypeByte   byte        //How is the vote being cast
+	Cost2Vote        types.Coins //Cost to vote
+	Cost2CreateIssue types.Coins //Cost to create a new issue
```

Comment:
Yes, that was the inspiration for using cost. Will replace with fee. Also, I think it makes sense to include a voting "fee" (hence the name pay to vote) but I've been conceptualizing it like you are paying an assigned vote token and not a regular currency per-se. Along these lines within the tests, the fees are of the types "votingTokens" and "issueTokens" (token allowing you to create an issue). Let me know your thoughts, I'm open to suggestions